ownCloud/Import/ImportFilesController.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  ImportFilesController.swift
+//  ownCloud
+//
+//  Created by Matthias Hühne on 10.07.19.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+/*
+* Copyright (C) 2019, ownCloud GmbH.
+*
+* This code is covered by the GNU Public License Version 3.
+*
+* For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+* You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+*
+*/
+
+import UIKit
+import ownCloudSDK
+
+class ImportFilesController: NSObject {
+
+	// MARK: - Instance variables
+	var url: URL
+
+	// MARK: - Init & Deinit
+	init(url: URL) {
+		self.url = url
+	}
+
+}
+
+extension ImportFilesController {
+
+	func accountOrImportUI() -> Bool {
+		let bookmarks : [OCBookmark] = OCBookmarkManager.shared.bookmarks as [OCBookmark]
+
+		if bookmarks.count > 1 {
+			let moreViewController = self.cardViewController(for: url)
+			if let wd = UIApplication.shared.delegate?.window {
+				let vc = wd!.rootViewController
+				if let navCon = vc as? UINavigationController, let topVC = navCon.visibleViewController {
```

Comment:
Please use `window` instead of `wd`, `viewController` instead of `vc`, `navigationController` instead of `navCon` and `topViewController` instead of `topVC` to match the code style of the rest of the code base.